### PR TITLE
Little updates

### PR DESCRIPTION
**Pull request by akhatmullin**
I say about commented code in files again. :)
Also I added two libraries: MBProgressHUD and AlecrimCoreData

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,12 +10,19 @@ Let's use the apple's official api design guidelines https://swift.org/documenta
...
+sonar.exclusions=Artifacts/\*\*,Application/ProjectName/Assets.xcassets/\*\*,Application/Pods/\*\*
+
 ## Best Practices
 
 Let's use swift community best practices https://github.com/schwa/Swift-Community-Best-Practices
 
 Also take a look at our [commit style](commit-style.md)
 
+Don't save commented code in your files. You can use Version Editor in Xcode to see the file's history.
+
```
Comment (zummenix) [inline README.md:25]: I believe this is mentioned somewhere in guides, but there is nothing wrong to mention it one more time )

**Comment by zummenix** [comment]
Thanks!

File: README.md (hunk 1 above)
Comment (akhatmullin) [reply README.md:25]: I haven't found it anywhere.

**Comment by akhatmullin** [comment]
You're welcome!

**Pull request merged**